http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -156,22 +159,18 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       samResourceId <- F.delay(AppSamResourceId(UUID.randomUUID().toString, req.accessScope))
 
-      // Look up the original email in case this API was called by a pet SA
-      originatingUserEmail <- authProvider.lookupOriginatingUserEmail(userInfo)
-
       // Retrieve parent workspaceId for the google project
       parentWorkspaceId <- samService.lookupWorkspaceParentForGoogleProject(userInfo.accessToken.token, googleProject)
 
       notifySamAndCreate = for {
-        _ <- authProvider
-          .notifyResourceCreated(samResourceId, originatingUserEmail, googleProject)
-          .handleErrorWith { t =>
-            log.error(ctx.loggingCtx, t)(
-              s"Failed to notify the AuthProvider for creation of kubernetes app ${googleProject.value} / ${samResourceId.asString}"
-            ) >> F.raiseError[Unit](t)
-          }
+        _ <- samService.createResource(userInfo.accessToken.token,
+                                       samResourceId,
+                                       Some(googleProject),
+                                       None,
+                                       Map("creator" -> SamPolicyData(List(userEmail), List(SamRole.Creator)))
```

Comment:
I think this creates a `kubernetes-app-shared`? Sam config has the ownerRoleName as `owner`; the SamRole might need to change?

Reply:
Oh great catch! I think you're right. Hopefully that will be caught by my BEE testing.

Reply:
Oh tricky. It actually works for `kubernetes-app-shared` with the current code because the `owner` and `user` policies are both inherited from the workspace, and the workspace owner is always the person who creates shared apps. Still I will fix this code (and add some comments)..